recipes/museval/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "museval" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2e1334dd3367dea562c1902515f6d55e87dc3072cfc411c616ac6f0a48d31d5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # this project need libsndfile, however, there is no such package for win platform
+  skip: true  # [win]
```

Comment:
If it's `noarch: python` and Windows fails due to missing dependencies that is okay as the package will only be built on Linux.